common/src/main/java/com/genexus/CommonUtil.java

```diff
@@ -1378,6 +1378,8 @@ public static Date dtadd(Date date, int seconds)
 
 	public static Date dtaddms(Date date, double seconds)
```

Comment:
dtaddms does not mean DateTime Add Milliseconds?
Why the double parameter is called "seconds"?

Reply:
No, dtaddms recieves seconds.